packages/client/hmi-client/src/components/widgets/tera-columnar-panel.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+	<main>
+		<slot />
+	</main>
+</template>
+
+<script setup lang="ts"></script>
+
+<style scoped>
+main {
+	display: grid;
+	grid-auto-flow: column;
+	grid-template-columns: repeat(auto-fit, minmax(0, 1fr));
+	overflow: hidden;
+	padding: 1rem;
```

Comment:
Maybe move this padding to `tera-drilldown` since it'll only be used there. That way you don't have to do a `padding: 0` in tera-model-description for example

Reply:
good idea, added!